reggie_config/init.yaml

```diff
@@ -3,20 +3,25 @@
 {%- set event_year = __grains__['event_year'] -%}
 {%- set event_tag = __grains__.get('event_tag', '') -%}
 {%- set private_ip = __grains__['ip4_interfaces'][__grains__.get('private_interface', 'eth1')][0] -%}
+{%- if event_tag -%}
+  {%- set tag_match = 'G@event_tag:' ~ event_tag -%}
+{%- else -%}
+  {%- set tag_match = 'not G@event_tag:*' -%}
+{%- endif -%}
```

Comment:
This block was pulled from ``loadbalancer.yaml``.  We made this change last year because we ran into the same issue - basically if you have two different staging servers, they can end up crossing wires.  What we saw last year was that the loadbalancer for the staging server would direct to the web servers for both the precon and onsite staging environments, because both environments matched ``G@env:staging and G@role:reggie and G@event_year:2019 and G@event:super``.

So we added the concept of an ``event_tag`` to disambiguate, which worked great.  However, when deploying this time I found that we kind of got lucky last year, because the wires can still get crossed for things like the ip of the database, etc.  They're not guaranteed to get crossed, and last year things just happened to work, which is why I say that we got lucky.

I fixed this by pulled the ``tag_match`` variable into this file and using it in a few more places.  In other words, this is just the same fix we made last year, but applied to more places.